Examples/RPC/rust/README.md

```diff
@@ -0,0 +1,17 @@
+# Rust HTTP RPC Sample
+
+This directory contains a sample Rust program that calls the Google Natural Language API using HTTP, RPC, and Protocol Buffers.
+
+## Protocol Buffer Support
+
+Message serialization requires Rust support code that is generated by protoc and the [Rust Protocol Buffers plugin](https://github.com/apple/swift-protobuf).
+
+## Authentication
+
+This sample authenticates with an API key that it reads from the `$GOOGLE_API_KEY` environment variable.
+
+## Instructions
+
+- Build and install `protoc-gen-rust` by running `cargo install protobuf` at the command line.
```

Comment:
AFAICT, the current version of the protobuf crate does not come with `protoc-gen-rust`. Currently need `cargo install protobuf --vers 1.0.24`

Reply:
Thanks, I've noticed that this had changed. I've been using this to get the plugin:
`cargo install protobuf-codegen`